auto-value-gson/src/test/java/com/ryanharter/auto/value/gson/AutoValueGsonExtensionTest.java

```diff
@@ -1366,13 +1366,22 @@
         + "}"
     );
 
+    // Collections default to empty
     assertAbout(javaSources())
         .that(Arrays.asList(nullable, source))
         .withCompilerOptions("-A" + COLLECTIONS_DEFAULT_TO_EMPTY + "=true")
         .processedWith(new AutoValueProcessor())
         .compilesWithoutError()
         .and()
         .generatesSources(expected);
+
```

Comment:
All properties in this test, including collection types, are wrapped with an `Optional`. Because the current implementation defaults optional values to the corresponding empty/absent value, no distinction is made between `Optional`-wrapped collection types and any other `Optional`-wrapped type. To confirm that this is the case, this test is run both with and without defaulting collection-typed properties to empty.